llama.cpp

```diff
@@ -12390,7 +12390,7 @@ struct llm_tokenizer_bpe {
                 const auto token = vocab.token_to_id.find(str);
                 // U+000A -> Ċ -> [UNK] (line feed, use [SEP] instead)
                 // U+0020 -> Ġ -> [UNK] (space, use U+3000 instead)
```

Comment:
Unicodeエスケープするとわかりにくくなりそうなので、以下のようにすると良さそうです！
```suggestion
                // U+000A -> U+010A (Ċ) -> [UNK] (line feed, use [SEP] instead)
                // U+0020 -> U+0120 (Ġ) -> [UNK] (space, use U+3000 instead)
```

Reply:
変更しましたー